File: src/main/java/com/example/demo/user/profile/UserProfileRoutes.java
package com.example.demo.user.profile;

import com.example.demo.user.auth.UserAuthRoutes;

public final class UserProfileRoutes {

    public static final String BASE_PATH = UserAuthRoutes.BASE_PATH;
    public static final String ME_ROUTE = "/me";
    public static final String ME_PATH = BASE_PATH + ME_ROUTE;

    private UserProfileRoutes() {
    }
}

File: src/main/java/com/example/demo/user/profile/UserProfileRoutes.kt
package com.example.demo.user.profile

import com.example.demo.user.auth.UserAuthRoutes

object UserProfileRoutes {
    const val BASE_PATH: String = UserAuthRoutes.BASE_PATH
    const val ME_ROUTE: String = "/me"
    const val ME_PATH: String = BASE_PATH + ME_ROUTE
}
